gradle/findbugs-exclude.xml

```diff
@@ -95,4 +100,74 @@
         <Class name="org.apache.kafka.connect.storage.OffsetStorageWriter"/>
         <Bug pattern="IS2_INCONSISTENT_SYNC"/>
     </Match>
+
+    <Match>
+        <!-- Suppress warnings about comparing a config string to
+             ConfigDef.NO_DEFAULT_VALUE, and calling new String(String)  These are intentional in
+             ConfigDef. -->
+        <Package name="org.apache.kafka.common.config"/>
```

Comment:
I merged Ewen's PR this morning that changes this code. I don't think we need this any more.